### DISCUSSION TITLE
Some doubts I had when using.

### DISCUSSION THREAD

**Discussion by Thiagobhmg** [inspired-futures/ipade#12] [Q&A]
First congratulations on the project. I thought it was fantastic.

I had some doubts.

This project is using a Jitsi server so the limit of people in a room would be 100 people?

Is it possible to direct to another personal Jitsi server to add more people than the limit of the free Jitsi service?

I did not find where to change the default language to be loaded in the code, could you indicate?

Once again the project is very good!

**Comment by deleolajide** [comment]
Thank yo for the kind words. For answers to your questions, please vist https://jitsi.github.io/handbook/

> This project is using a Jitsi server so the limit of people in a room would be 100 people?

No, there is no limit on number of people. There is a rough 1000 HD video streams limit on the SFU server because of bandwidth. constraints with 1GB network connections. 1000 streams will give you a single conference of 32 active video streams. Everyone else will be muted. It can also give you 500 peer-to-peer video calls. Each conference uses (N x N-1) streams.

> I did not find where to change the default language to be loaded in the code, could you indicate?

Read the Jitsi documentation for config.js
